src/mbed_tools/build/_internal/templates/mbed_config.tmpl

```diff
@@ -7,6 +7,14 @@
 set(MBED_TOOLCHAIN "{{toolchain_name}}" CACHE STRING "")
 set(MBED_TARGET "{{target_name}}" CACHE STRING "")
 set(MBED_CPU_CORE "{{core}}" CACHE STRING "")
+set(MBED_C_LIB "{{c_lib}}" CACHE STRING "")
+set(MBED_PRINTF_LIB "{{printf_lib}}" CACHE STRING "")
+
+list(APPEND MBED_TARGET_SUPPORTED_C_LIBS {% if toolchain_name.lower() in supported_c_libs -%} {% for supported_c_lib in supported_c_libs[toolchain_name.lower()] %}
```

Comment:
Should we move the logic to get the list of supported c libs into the python module?

Reply:
Is [this](https://github.com/ARMmbed/mbed-tools/compare/cd1d24ce4340bf582e545f3c869b1f0e87a2657b..aeec090577452b406c64135aff8a1db84904ec36) ok?

Reply:
Much better. Thanks.